test/function/string-split.yml

```diff
@@ -0,0 +1,51 @@
+name: "string::split() function"
+
+features:
+- namespaces
+- experimentalFunctions
+
+tests:
+- name: Basic
+  query: |
+    string::split(~str~, ~sep~)
+  tests:
+  # Basic
+  - variables:
+      str: '"a,b,c"'
+      sep: '","'
+    result: [a, b, c]
+  # Separator at ends
+  - variables:
+      str: '",a,b,c,"'
+      sep: '","'
+    result: ["", a, b, c, ""]
+  # empty string
+  - variables:
+      str: '""'
+      sep: '","'
+    result: []
+  # Just separator
+  - variables:
+      str: '","'
+      sep: '","'
+    result: ["", ""]
+  # Empty separator
+  - variables:
+      str: '"møøse"'
+      sep: '""'
+    result: ["m", "ø", "ø", "s", "e"]
```

Comment:
An example with emoji (or beyond U+FFFF) would be good to catch JavaScript bugs. We _don't_ want the JavaScript behavior where `split` splits up Unicode characters.